src/main/res/values/strings.xml

```diff
@@ -32,6 +32,7 @@
     <string name="connectionError">Internet connection problem</string>
     <string name="errorDownload">Download error</string>
     <string name="errorDownloadInsufficientSpace">Cannot download : Insufficient space</string>
+    <string name="errorFileNotFound">An error occurred when opening this file</string>
```

Comment:
Wouldn't "An error **has** occurred" show that the error **just** occurred right now and therefore be a bit more precise?

Reply:
My take would be "An error has occured while opening this file"

Reply:
occurred?